openshift/Makefile.ci

```diff
@@ -1,7 +1,17 @@
 # Makefile.ci holds make directives called by the CI jobs for this repo.
+TARGET = kubefed
+GOTARGET = sigs.k8s.io/$(TARGET)
```

Comment:
This should be `github.com/openshift/$(TARGET)`

Reply:
It expects a value which is defined for `canonical_go_repository` parameter in config.yaml file. The value you're suggesting doesn't pass the ci-operator tests locally.